python/cm/bundle.py

```diff
@@ -265,11 +265,18 @@ def update_obj(dest, source, fields):
         setattr(dest, f, getattr(source, f))
 
 
+def extend_role(name, children):
+    role = Role.objects.get(name=name)
+    for child_role in children:
```

Comment:
We don't need a loop here, we can add all the children like this `role.child.add(*children)`

Reply:
fixed